soc/wch/ch32v/ch32v00x/CMakeLists.txt

```diff
@@ -3,8 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_sources(
-  soc_irq.S
-  vector.S
+  ../common/qingke_v2x/soc_irq.S
```

Comment:
I don't think a directory should refer to a different directory. I can't find any examples of this in the other socs.

As soc_irq.S is fairly trivial, shall we just denormalise it across all SOCs?